src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java

```diff
@@ -114,14 +115,17 @@ private Layer convertToLayer(Collection collection, boolean includeSource, UUID
                 .group(collection.getGroup() != null ? collection.getGroup().getName() : null)
                 .legend(collection.getStyleRule() != null ?
                         JsonUtil.readObjectNode(collection.getStyleRule(), Legend.class) : null)
+                .style(collection.getStyle() != null ?
+                        JsonUtil.readObjectNode(collection.getStyle(), Style.class) : null)
```

Comment:
Was it a requirement to parse JSON on BE side instead of FE?

Reply:
I don't know how it was agreed, but for legend it was done before. So I did the same way.

Reply:
Then probably you should ask FE if it's okay for them